### PR TITLE
Outlined all three prospective test commands.

### PR DESCRIPTION
**Pull request by BrandyMcKinney**
## Proposed changes

I edited the README file by adding two more tests under the test section. Also, added  a script command ```test:all```  that will run both unit and end-to-end tests.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Refactor (found a better way to write the code and accomplish the same task)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by gatorjuice** [comment]
@BrandyMcKinney you can continue to commit and add to this branch and I will review the whole thing with you when it's ready. I just you aren't waiting for review like you thought you had to stop and get that from me.

**Comment by BrandyMcKinney** [comment]
Hey Jamie!
It says I can reply directly to this email...I guess I'll find out!
Oh no, I'm not waiting for your review to proceed. It's all good! I will
finish the test issue.  I have to organize my time for learning (I have
time blocked out for different things --practice/reading/projects..etc).
Still figuring out the best schedule. The plan is to be back on
alumni_portfolio_vue tomorrow evening after work!

File: .github/workflows/test.yml
Comment (gatorjuice) [inline .github/workflows/test.yml:23]: We don't need to call this command here as part of CI because both sets of tests are already called individually in the previous steps, so this would be redundant.
```suggestion
```

File: .github/workflows/test.yml
Comment (gatorjuice) [inline .github/workflows/test.yml:38]: Your linter likely changed this when you opened this file in your editor. I don't mind this changing but considering that since we don't need the `yarn run:all` command in this file, we should probably remove all changes to this file so it doesn't even show up in this PR. In reality though you should look into configuring your linter to play by the organizations rules and styles so you don't introduce out of scope changes in your PR's
```suggestion
          wait-on: 'http://localhost:8080'
```

File: README.md
Comment (gatorjuice) [inline README.md:20]: thank you!

File: README.md
Comment (gatorjuice) [inline README.md:44]: A few things with this section. Can you add a description for just running cypress e2e tests? And, concurrently is not the right word here since the test suites don't run at the same time as a result of this command. 
```suggestion
### Run unit tests and then end-to-end tests
```

File: package.json
Comment (gatorjuice) [inline package.json:13]: You can reference other script calls from within a yarn command and you can separate commands with `&&` to perform multiple. So if I had `yarn foo` and `yarn bar` I could design a third command `yarn foobar` that executes `yarn foo && yarn bar`.
```suggestion
    "test:all": "<what goes here?>"
```

**Review by gatorjuice — CHANGES_REQUESTED** [review]
Almost there! Just a slight adjustment to the command thinking and you're there.

File: .github/workflows/test.yml
Comment (gatorjuice) [inline .github/workflows/test.yml:23]: We don't need to call these commands here as part of CI because this section of the workflow is ONLY responsible for calling the unit tests. Look elsewhere in the file to see where the e2e tests are configured. No changes are needed in this file.
```suggestion
```

**Review by gatorjuice — CHANGES_REQUESTED** [review]

**Comment by gatorjuice** [comment]
You can see how adding the commands to `test.yml` actually broke your test run. It's important to pay attention to these check on every PR you create.

Screen Shot 2022-01-08 at 9 12 46 AM
Screen Shot 2022-01-08 at 9 13 26 AM

File: .github/workflows/test.yml
Comment (BrandyMcKinney) [reply .github/workflows/test.yml:38]: Yes, is it my linter.  Each time I save the code it reverses back to double quotes. I'll fix this in my editor.

File: README.md
Comment (BrandyMcKinney) [reply README.md:44]: You are absolutely right! Thank you. I will correct.

File: package.json
Comment (BrandyMcKinney) [reply package.json:13]: Ahhh!! I initially had this but couldn't figure out how to separate the commands. I used a comma, double quotes with a comma, etc. Completely bypassed using a simple &&.  So used the "pre" workaround. Thank you!

File: .github/workflows/test.yml
Comment (BrandyMcKinney) [reply .github/workflows/test.yml:23]: Deleted these lines of code.

File: package.json
Comment (gatorjuice) [inline package.json:12]: wonderful, this will suffice and give us the freedom to run the tests with different flags perhaps in the future. We might not want got show coverage for example when the tests run back to back because that coverage report just gets swallowed by the output.

you could have also done...
```suggestion
    "test:all": "yarn test:unit && yarn test:e2e",
```

I'll approve and leave it to you to decide but it's fine as is, I just want you to know your options.

**Review by gatorjuice — APPROVED** [review]
Nice work.

**Comment by gatorjuice** [comment]
@BrandyMcKinney go ahead and merge.

File: package.json
Comment (BrandyMcKinney) [reply package.json:12]: Thank you, Jamie!  Also, I've made note of the options.

**Pull request merged**
